### PR TITLE
🐛 FIX: parsing of unicode ordinals

### PR DESCRIPTION
**Pull request by hukkinj1**
Closes https://github.com/executablebooks/markdown-it-py/issues/80

This seems like a porting issue to me: Javascript needs two string characters for codepoints larger than `0xFFFF` but Python 3 can represent any unicode codepoint in one character.

This PR removes the handling that splits a large codepoint to two chars.

### CODE REVIEW

**Comment by hukkinj1** [comment]
@chrisjsewell can you think of any other places in the codebase where this Javascript string (string of UTF-16 code units) versus Python3 string (string of Unicode codepoints) difference could cause issues.

I just realized we unintentionally fixed one instance of this in https://github.com/executablebooks/markdown-it-py/pull/70 :smile: The old entity data seemed to contain UTF-16 code units.

**Comment by chrisjsewell** [comment]
Not that I know of, I'll have a quick look later

**Comment by lesteve** [comment]
Just a gentle reminder about this one ;-). I completely understand if you have higher priorities of course.

**Review by chrisjsewell — CHANGES_REQUESTED** [review]

File: markdown_it/common/utils.py [hunk 1]
```diff
@@ -92,15 +92,7 @@ def isValidEntityCode(c):
...
-def fromCodePoint(c):
-
-    if c > 0xFFFF:
-        c -= 0x10000
-        surrogate1 = 0xD800 + (c >> 10)
-        surrogate2 = 0xDC00 + (c & 0x3FF)
-
-        return "".join(map(chr, [surrogate1, surrogate2]))
-
+def fromCodePoint(c: int) -> str:
```
Comment (chrisjsewell) [inline markdown_it/common/utils.py:95]:
```suggestion
def fromCodePoint(c: int) -> str:
   """Convert ordinal to unicode.
   
   Note, in the original Javascript two string characters were required,
   for codepoints larger than `0xFFFF`.
   But Python 3 can represent any unicode codepoint in one character.
   """
```

File: markdown_it/common/utils.py [hunk 2]
```diff
@@ -92,15 +92,13 @@ def isValidEntityCode(c):
...
+   """Convert ordinal to unicode.
+   
+   Note, in the original Javascript two string characters were required,
+   for codepoints larger than `0xFFFF`.
+   But Python 3 can represent any unicode codepoint in one character.
+   """
     return chr(c)
```
Comment (chrisjsewell) [inline markdown_it/common/utils.py:102]:
```suggestion
    """Convert ordinal to unicode.

    Note, in the original Javascript two string characters were required,
    for codepoints larger than `0xFFFF`.
    But Python 3 can represent any unicode codepoint in one character.
    """
    return chr(c)
```

**Review by chrisjsewell — APPROVED** [review]

**Pull request merged**